### PR TITLE
Version banner + PropfilePopup fixes

### PR DESCRIPTION
**Pull request by jrainville**
Fixes the version banner appearing for no reason + at the wrong location. This also fixes the padding of the profile icon on the bottom left

Also fixes the profile popup by hooking it to the new backend properties. The only missing part for that popup are the community ones because there are still worked on

### CODE REVIEW

File: ui/app/AppMain.qml [hunk 1]
```diff
@@ -35,7 +35,7 @@ Item {
     // set from main.qml
     property var sysPalette
     // Not Refactored Yet
-    property var newVersionJSON: "" //JSON.parse(utilsModel.newVersion)
+    property var newVersionJSON: ({}) //JSON.parse(utilsModel.newVersion)
```
Comment (saledjenic) [inline ui/app/AppMain.qml:38]: @jrainville as a part of this ticket we should move code which is currently placed in the file `src/app/REMOVE-utilsView/view` to the `mainModule` (`QtProperty[string] newVersion` should be part of the `src/app/modules/main/view.nim`) and used in this line like: 
```
property var newVersionJSON: mainModule.newVersion
```

Async task `CheckForNewVersionTaskArg` for fetching new version from remote may be part of `about` service (if you want you may add kind of a `global` service, but I think having this as a part of `about` service is just fine) and have `getNewAppVersion` method, and you may just call it from a `controller` of the main module.

**Review by saledjenic — CHANGES_REQUESTED** [review]
@jrainville what is done so far is good, but I would add this as a part of this PR to have that completed here (nothing else is blocking us to do it here).

File: ui/app/AppMain.qml [hunk 2]
```diff
@@ -347,177 +332,197 @@ Item {
...
 
-        appView: StackLayout {
-            id: appView
+        appView: ColumnLayout {
             anchors.fill: parent
-            currentIndex: {
-                if(mainModule.activeSection.sectionType === Constants.appSection.chat) {
-                    return Constants.appViewStackIndex.chat
+
+            ModuleWarning {
```
Comment (borismelnik) [inline ui/app/AppMain.qml:338]: `ModuleWarning` logically some kind of `Module` or `Modal`?

Comment (jrainville) [reply ui/app/AppMain.qml:338]: It's kinda like a banner actually. I can rename it if you want

**Comment by jrainville** [comment]
@saledjenic updated. 

Now also depends on https://github.com/status-im/status-lib/pull/138

**Review by saledjenic — APPROVED** [review]

**Pull request merged**